build.gradle

```diff
@@ -11,7 +11,7 @@ repositories {
 }
 
 group = "pro.civitaspo"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
I do this change when releasing, so could you remove this change?

Reply:
Reverted. Please confirm

Reply:
thx